examples/api/test/material/floating_action_button/floating_action_button.0_test.dart

```diff
@@ -16,11 +16,11 @@ void main() {
     expect(find.byType(FloatingActionButton), findsOneWidget);
     expect(find.byIcon(Icons.navigation), findsOneWidget);
 
-    final Finder materialButtonFinder = find.byType(RawMaterialButton);
-    RawMaterialButton getRawMaterialButtonWidget() {
-      return tester.widget<RawMaterialButton>(materialButtonFinder);
-    }
-    expect(getRawMaterialButtonWidget().fillColor, Colors.green);
-    expect(getRawMaterialButtonWidget().shape, const CircleBorder());
+    final Material material = tester.widget(find.descendant(
```

Comment:
There may be tests like this in google that need to be migrated, like the ToggleButton change. I am running internal tests now. :)